lib/bundler/cli/install.rb

```diff
@@ -185,5 +181,13 @@ def print_post_install_message(name, msg)
       Bundler.ui.confirm "Post-install message from #{name}:"
       Bundler.ui.info msg
     end
+
+    def check_for_options_conflicts
```

Comment:
Yeah I agree. There is an `InvalidOption` error that we could raise, and we could add a handler for that to `log_error` in `Bundler::FriendlyErrors` that would always log the message and also output `invalid command usage` ?